Messenger/AppDelegate.mm

```diff
@@ -698,6 +698,11 @@ - (void)windowDidBecomeKey:(NSNotification*)notification {
   [self evaluateJavaScript:@"try { (typeof MacMessenger != 'undefined') && MacMessenger.focusComposer(); } catch(_) {}"];
 }
 
+- (void)windowDidResignKey:(NSNotification *)notification {
+  // Make sure a blur event is triggered when the app is on another workspace
```

Comment:
Right, this is called every time it goes into the background, the comment should clarify that the manual triggering is only needed for the case when it's in a different workspace. My intention was to call it every time it loses focus, just a bad comment